solr/solr-ref-guide/modules/configuration-guide/pages/solr-modules.adoc

```diff
@@ -17,32 +17,35 @@
 // specific language governing permissions and limitations
 // under the License.
 
-Solr modules are addon Solr plugins that are not part of solr-core, but officially maintained
+Solr Modules are addon Solr plugins that are not part of solr-core, but officially maintained
 by the Solr project. They provide well-defined features such as the "extracting" module which lets
 users index rich text documents with Apache Tika. A single module can contain multiple Plugins.
 Modules were earlier known as "contribs".
 
-Each module produces a separate `.jar` file in the build, and additional dependencies required by
-each module are also packaged with the module. This helps keep the main core of Solr small and lean.
+Each module produces a separate `.jar` file in the build, packaged in the module's `lib/` directory.
+All additional dependencies required by the module, and not provided by Solr core, are also packaged there.
+This helps keep the main core of Solr small and lean.
 
 == Installing a module
 
 The easiest way to enable a module is to list the modules you intend to use either in the
 system property `solr.modules` or in the environment variable `SOLR_MODULES` (e.g. in `solr.in.sh`
-or `solr.in.cmd`). You can also add a `<str name="modules">` tag to
+or `solr.in.cmd`). You can also add a `<str name="modules">` tag to your
 xref:configuration-guide:configuring-solr-xml.adoc[solr.xml]. The expected value is a comma separated list
-of module names, e.g. `SOLR_MODULES=extracting,ltr`. This way of adding modules will add
-them to the shared class loader, making them available for every collection in Solr.
+of module names, e.g. `SOLR_MODULES=extracting,ltr`. This will make the functionality of configured Modules
+available to every collection in Solr.
 
-You can also specify the modules when using the Solr CLI to start Solr:
+You can also specify the modules to include when using the Solr CLI to start Solr:
 
 [source,bash]
 ----
 bin/solr start -e techproducts -Dsolr.modules=scripting
 ----
 
-If you only wish to enable a module for a single collection, you may add `<lib>` tags to `solrconfig.xml`
+NOTE: If you only wish to enable a module for a single collection, you may add `<lib>` tags to `solrconfig.xml`
 as explained in xref:configuration-guide:libs.adoc[Lib Directories].
+Collection-level plugins will work if the module is enabled either per collection (`<lib>`) or for the whole Solr node.
+Cluster-level plugins will not work when using the `<lib>` option, they must be enabled for the entire Solr node, as described above.
```

Comment:
```suggestion
Node-level plugins such as those specified in `solr.xml` will not work when using the `<lib>` option in `solrconfig.xml` because configsets configure collections, not the node.  They must be enabled for the entire Solr node, as described above.
```

I'm not sure we speak of "Cluster level"; node is clearer.